lib/tags.sql

```diff
@@ -0,0 +1,9 @@
+-- Check if 'tags' table exists already; create if not
+CREATE TABLE IF NOT EXISTS tags (
+  id INTEGER PRIMARY KEY,
+  name VARCHAR(50) NOT NULL,
+  active BOOLEAN NOT NULL
+);
+
+-- Delete all entries of the tags table
+TRUNCATE tags;
```

Comment:
It might be safer to delete completely the table and to recreate it. This will allow us to update the table structure